app/controllers/idv/personal_key_controller.rb

```diff
@@ -63,9 +69,7 @@ def finish_idv_session
 
       irs_attempts_api_tracker.idv_personal_key_generated
 
-      if idv_session.address_verification_mechanism != 'gpo'
-        flash.now[:success] = t('idv.messages.confirm')
-      end
+      flash.now[:success] = t('idv.messages.confirm')
```

Comment:
Should we show the confirm message if someone is under threatmetrix review? That's probably a different PR.

Reply:
Hmm, that is an interesting thought. I think that is a different change though. This maintains the existing behavior.